packages/ckeditor5-dev-release-tools/tests/tasks/updateversions.js

```diff
@@ -0,0 +1,143 @@
+				'../utils/getpackagejson': stubs.getPackageJson,
+				'glob': { globSync: stubs.sync },
+				'@ckeditor/ckeditor5-dev-utils': { tools: { shExec: stubs.shExec } },
+				'semver': { gt: stubs.gt, valid: stubs.valid }
+			} );
+		} );
+
+		afterEach( () => {
+			sandbox.restore();
+		} );
+
+		it( 'should update the version field in all found packages including the root package', () => {
+			stubs.sync.returns( [
+				'/ckeditor5-dev/packages/package1/package.json',
+				'/ckeditor5-dev/packages/package2/package.json',
+				'/ckeditor5-dev/packages/package3/package.json',
+				'/ckeditor5-dev/package.json'
+			] );
+
+			updateVersions( { version: '1.0.1', packagesDirectory: 'packages' } );
+
+			expect( stubs.sync.firstCall.args[ 0 ] ).to.deep.equal( [ 'package.json', 'packages/*/package.json' ] );
+			expect( stubs.outputJsonSync.callCount ).to.equal( 4 );
+			expect( stubs.outputJsonSync.getCall( 0 ).args[ 0 ] ).to.contain( '/ckeditor5-dev/packages/package1/package.json' );
+			expect( stubs.outputJsonSync.getCall( 3 ).args[ 0 ] ).to.equal( '/ckeditor5-dev/package.json' );
+			expect( stubs.outputJsonSync.firstCall.args[ 1 ] ).to.deep.equal( { version: '1.0.1' } );
+		} );
+
+		it( 'should update the version field in the root package when packagesDirectory is not provided', () => {
+			stubs.sync.returns( [ '/ckeditor5-dev' ] );
+
+			updateVersions( { version: '1.0.1' } );
+
+			expect( stubs.sync.firstCall.args[ 0 ] ).to.deep.equal( [ 'package.json' ] );
+			expect( stubs.outputJsonSync.callCount ).to.equal( 1 );
+			expect( stubs.outputJsonSync.firstCall.args[ 0 ] ).to.contain( '/ckeditor5-dev' );
+			expect( stubs.outputJsonSync.firstCall.args[ 1 ] ).to.deep.equal( { version: '1.0.1' } );
+		} );
+
+		it( 'should throw an error when the version is already in use', () => {
+			stubs.getPackageJson.returns( { version: '1.0.0', name: 'stub-package' } );
+			stubs.shExec.returns( '' );
+
+			expect( () => updateVersions( { version: '1.0.1' } ) )
+				.to.throw( Error, 'Provided version 1.0.1 is already used in npm by stub-package' );
+		} );
+
+		it( 'should not throw an error when version is not in use', () => {
+			stubs.shExec.throws( new Error( 'is not in this registry' ) );
+			stubs.getPackageJson.returns( { version: '1.0.0', name: 'stub-package' } );
+
+			expect( () => updateVersions( { version: '1.0.1' } ) ).to.not.throw( Error );
+		} );
+
+		it( 'should throw an error when checking the version availability check throws error', () => {
+			stubs.shExec.throws( new Error( 'custom error' ) );
+			stubs.getPackageJson.returns( { version: '1.0.0', name: 'stub-package' } );
+
+			expect( () => updateVersions( { version: '1.0.1' } ) ).to.throw( Error, 'custom error' );
+		} );
+
+		it( 'should accept a "0.0.0-nightly-123*" version for nightly releases', () => {
+			updateVersions( { version: '0.0.0-nightly-123.0' } );
+
+			expect( stubs.outputJsonSync ).to.be.called;
+		} );
```

Comment:
[As I mentioned in the previous review](https://github.com/ckeditor/ckeditor5-dev/pull/870#discussion_r1187781952), it won't work, and this test checks nothing.

I integrated the script and passed `0.0.0-nightly-20220509.0` as the version value.

```
Error: Provided version 0.0.0-nightly-20220509.0 must be greater than 37.1.0.
    at checkVersionGreaterThanOldVersion (/Users/pomek/Projects/ckeditor/ckeditor5/external/ckeditor5-dev/packages/ckeditor5-dev-release-tools/lib/tasks/updateversions.js:72:9)
    at Object.updateVersions (/Users/pomek/Projects/ckeditor/ckeditor5/external/ckeditor5-dev/packages/ckeditor5-dev-release-tools/lib/tasks/updateversions.js:37:2)
    at /Users/pomek/Projects/ckeditor/ckeditor5/scripts/release/prepare-packages.js:44:15
```